tests/test_repo_info_extractor.py

```diff
@@ -0,0 +1,18 @@
+import json
+import unittest
+from pprint import pprint
+
+from sim_dev_search.utils.repos_info_extractor import ReposInfoExtractor
+
+
+class RepoExtractorTestCase(unittest.TestCase):
```

Comment:
I recommend to simplify tests
* check number of developers
* number of files per developer
* sum of lines
* etc